### PR TITLE
add yaml json and rawdata json

### PR DESCRIPTION
**Pull request by wuchang0201**
add yaml json and rawdata json，Users can directly jump to the corresponding interface through the link to view detailed pod and node information

### CODE REVIEW

**Review by d3c3mber — APPROVED** [review]

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 1]
```diff
@@ -696,7 +697,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "NodeYaml",
-                        "url": "http://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpodyaml?name=${__data.fields[\"NodeName\"]}"
+                        "url": "http://localhost:9097/d/nodeyamllinks/nodeyaml?orgId=1&var-nodeName=${__data.fields[\"NodeName\"]}"
```
Comment (linuzb) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:700]: Don't use hard-coded jump links. Perhaps you could use the url's hostname variable as the hostname of the jump link

Comment (wuchang0201) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:700]: We have recorded an issue and will resolve it later

Comment (d3c3mber) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:700]: we could use the same URL for grafana dashboard access and for node yaml/pod yaml link redirection

**Review by d3c3mber — APPROVED** [review]

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml [hunk 2]
```diff
@@ -671,7 +672,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "yaml",
-                        "url": "http://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpodyaml?uid=${__data.fields[\"PodUID\"]}"
+                        "url": "/d/podyamllinks/podyaml?orgId=1&var-uid=${__data.fields[\"PodUID\"]}"
```
Comment (d3c3mber) [inline deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:675]: @linuzb PTAL

**Review by d3c3mber — APPROVED** [review]

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml (hunk 2 above)
Comment (wuchang0201) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:675]: In Grafana, you don't need a domain and can jump over directly

**Review by hnhbwlp — APPROVED** [review]
LGTM

File: deploy/helm/lunettes/templates/grafana/grafana-cm.yaml (hunk 2 above)
Comment (d3c3mber) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:675]: @linuzb currently "/d" supports link redirection

Comment (linuzb) [reply deploy/helm/lunettes/templates/grafana/grafana-cm.yaml:675]: SGTM

**Pull request merged**